src/main/webapp/script.js

```diff
@@ -107,7 +107,7 @@ function handleTextResponse(response) {
  */
 function loadCalendar() {
   fetch("/user")
-    .then(handleTextResponse)
+    .then(getJsonIfOk)
```

Comment:
if this makes `handleTextResponse` unused, you could also remove it again

Reply:
It is used in the `schedule()` function for now.